test/pylib/util.py

```diff
@@ -33,15 +35,15 @@ def process(self, msg, kwargs):
 T = TypeVar('T')
 
 
-def unique_name():
+def unique_name(unique_name_prefix = 'test_'):
     if not hasattr(unique_name, "last_ms"):
         unique_name.last_ms = 0
     current_ms = int(round(time.time() * 1000))
     # If unique_name() is called twice in the same millisecond...
     if unique_name.last_ms >= current_ms:
         current_ms = unique_name.last_ms + 1
     unique_name.last_ms = current_ms
-    return unique_name_prefix + str(current_ms)
+    return unique_name_prefix + ''.join(random.choice(string.ascii_lowercase) for _ in range(5)) + str(current_ms)
```

Comment:
Nit: I think I'd rather put the random part at the end rather than in the middle. I can imagine that somebody would like to sort or navigate through the generated identifiers somehow. Putting the timestamp right after the prefix would make it easier as the alphabetical ordering would correspond to the time ordering.

Reply:
An underscore between the timestamp and the random part would also improve readability.